### PR TITLE
Removes cloning

### PR DESCRIPTION
**Pull request by alexkar598**
simple enough 
image

# obligatory improves rp
<details>
 <summary>
#######################
#####PR TEMPLATE#######
#######################</summary>

# General Documentation

### Intent of your Pull Request

Remove cloning

### Why is this change good for the game?

Improves RP

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Removes cloning, it removes cloning.

### What should players be aware of when it comes to the changes your PR is implementing?
Cloning is removed

### What general grouping does this PR fall under? 
Tools powergamers use

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
Anything that might make it look bad

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
I will be removing 1 cloning pod. I will also be removing 1 cloning computer and I will finally be removing 1 cloning scanner.

Oh and also the entire room yeah.

# Changelog
:cl:  
rscdel: Removed cloning. 
/:cl:
</details>

### CODE REVIEW

**Comment by ghost** [comment]
Exactly what yogstation needs right now

**Comment by bowlofcereal** [comment]
good PR, very well balanced and optimal for the server, will create RP

**Comment by ghost** [comment]
putting the 'rp' in 'pr'

**Comment by Reed0506** [comment]
M-M-MUH CLOOOOONNINNNNNNNNNNNNNGGGGGGGG!!!!!!!!!!!!!!!!

**Comment by Marmio64** [comment]
better than swissloaf PRs tbh

**Comment by Hadoje** [comment]
RP has already increased just by looking at this PR

**Review by jcatjmeow — DISMISSED** [review]
Fill out the goddam pr template smh smh

**Comment by Hadoje** [comment]
Actually increased RP by showing me that there's an IV drip on one of the beds below surgery in medbay

**Review by jcatjmeow — APPROVED** [review]

**Comment by maxion12345** [comment]
Jesus fuck he's turning into adam, where he doesn't explain anything about why he's doing his PR beyond "yeah bro it's an improvement"

**Comment by bowlofcereal** [comment]
you dont have to fill out the template when your PR is this good

by far the most balanced and best update to the entire game, we should just make him the host already, no, the CEO of byond, he is the most intelligent and extremely best coder in the entire fucking world

he just solved world hunger by doing this

**Comment by LoliconSlayer** [comment]
Literally Wuba/maxion12345:
image

**Review by UselessTheremin — APPROVED** [review]

**Review by patpol4 — APPROVED** [review]
Seems extreme but it is technically fine

**Comment by ghost** [comment]
whys this not merged, they did the mining point thing pretty fast w/o test merge, why not this
